Makefile

```diff
@@ -0,0 +1,43 @@
+# Variables
+DCURL_DIR=dcurl
+ROCKSDB_DIR=rocksdb
+ROCKSDB_BRANCH=5.18.fb
+ROCKSDB_VERSION=5.18.1
+
+.PHONY: all download dcurl rocksdb iri clean
+
+# Rules
+all: download dcurl rocksdb iri
+
+download:
+	if [ ! -d ${DCURL_DIR} ]; then \
+	    git clone https://github.com/DLTcollab/dcurl.git ${DCURL_DIR}; \
+	fi
+	if [ ! -d ${ROCKSDB_DIR} ]; then \
+	    git clone https://github.com/DLTcollab/rocksdb.git ${ROCKSDB_DIR}; \
+	fi
+
+dcurl:
+	$(MAKE) -C ${DCURL_DIR} BUILD_AVX=1 BUILD_JNI=1
+	cp ${DCURL_DIR}/build/libdcurl.so ./
```

Comment:
You don't really have to copy built shared library here.

Reply:
The command has been deleted.